class CheckResponseException(Exception):
    """Исключение для функции CheckResponse."""
    pass
